quic/s2n-quic-platform/src/message/msg/tests.rs

```diff
@@ -50,8 +66,13 @@ fn address_inverse_pair_test() {
 }
 
 #[test]
-#[cfg_attr(kani, kani::proof, kani::solver(cadical), kani::unwind(65))]
-#[cfg(any(not(kani), kani_slow))] // this test isn't able to terminate
+#[cfg_attr(
+    kani,
+    kani::proof,
+    kani::solver(cadical),
+    kani::unwind(65),
+    kani::stub(cmsg::decode, stubs::decode)
```

Comment:
Can we add a comment as to why this is ok to stub (the cmsg result isn't actually checked in this particular test)

Reply:
Added a comment.